bin/oneoff/back_populate_forms_hashed_email

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
+
+# This script populates the hashed_email column of the pegasus forms table.
+# This is based off of back_populate_contacts_hashed_email
+
+require File.expand_path('../../../pegasus/src/env', __FILE__)
```

Comment:
Yep, turns out this is necessary, it pulls in some stuff that this won't run without